no-computer.md

```diff
@@ -58,12 +68,17 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+This would be very focused on the language, especially since there doesn't have to be a computer involved at all.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I'm not entirely sure. Since not much like this has been done before to my knowledge, it could turn out super easy to describe game rules or (as I expect) it could potentially get quite hard once you really start to consider a lot of the different play styles that are out there, even for board games.
```

Comment:
I think the most difficult part of this would be deciding what basic kind of model you want to use. Are you going to model games as finite state machines, or just something with some kind of state where players make decisions? What kinds of decisions can players make? Can they only make decisions from some finite set of choices? If so it might be hard to model something like property negotiation in Monopoly. And so on and so forth. It might be more wise to limit scope to some particular kind of board game, or even to some even more specific set of board games and make the goal to be a language that works for all of those board games in the set, and slowly extend the set of board games that it works for.